File: abc/086/a/cpp/main.cpp
#include <iostream>

using namespace std;

bool isEven(int n)
{
    return n % 2 == 0;
}

int main()
{
    int a, b;
    cin >> a >> b;
    string output = isEven(a) || isEven(b) ? "Even" : "Odd";
    cout << output << endl;
}

File: abc/086/a/rust/main.rs
#!/usr/bin/env rust-script

//! ```cargo
//! [dependencies]
//! proconio = "0.4.5"
//! ```

use proconio::input;

fn is_even(n: i32) -> bool {
    n % 2 == 0
}

fn main() {
    input! {
        a: i32,
        b: i32
    }

    let output = if is_even(a) || is_even(b) {
        "Even"
    } else {
        "Odd"
    };
    println!("{}", output);
}
